pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -104,6 +104,12 @@ def _get_gateset(gates: List[str]) -> Set[OpType]:
     return gs
 
 
+def _flatten_registers(c: "Circuit") -> "Circuit":
+    c.remove_blank_wires()
+    c.rename_units({c.qubits[i]: Qubit("quantinuum", i) for i in range(len(c.qubits))})
```

Comment:
```suggestion
    c.rename_units({qb: Qubit("quantinuum", i) for i, qb in enumerate(c.qubits)})
```
Just a suggestion for style and efficiency, feel free to ignore.

Reply:
added

Reply:
done